src/main/java/dev/morling/jfrunit/JfrEventTest.java

```diff
@@ -23,7 +23,6 @@
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.spockframework.runtime.extension.ExtensionAnnotation;
 
-@ExtensionAnnotation(JfrEventTestSpockExtension.class)
```

Comment:
> @sbrannen Would please help me with reproducing your originally failing case in JUnit5 so I can better assure it works for you. Thanks in advance.

The removal of this prevents the build failure experienced in the JUnit 5 build.